### ISSUE TITLE
Select the space after a "failed" word

### ISSUE THREAD

**Issue by dmland** [mortenjust/cleartext-mac#60]
Right now, the document ends up with a bunch of abandoned spaces to the right of the text input pointer because Cleartext selects only the word, and not the space that triggered the _isSimple_ check. Writing would be much faster if the space was also selected, and we could just type a simpler word (and a space to check _it_) and keep on going.

**Comment by Serspring** [comment]
Working now.

**Comment by dmland** [comment]
Thanks! Well, that's what I get for only using the pre-built binary: 1.31 doesn't work as described.

I guess I'll either have to install a build environment on my El Capitan laptop or be patient.

**Comment by Serspring** [comment]
I uhh... Tried? 2 hours of my attempt at understanding this program and Swift and also programming on a school Mac, and not having the ability to use Swift 2.0. Sorry.

https://github.com/mortenjust/cleartext-mac/pull/61

**Comment by dmland** [comment]
Thanks @Serspring,

I'm in the same boat: I figured, "Hey, it's just another language…," but I'm stumped.

I would love for someone who _gets_ Swift to extend the selection to include the space, but it's beyond my ken at this point :-/.

PS: When you wrote "working now", I assumed that you meant "_it is_ working now," which is why I responded the way I did. I didn't realize that you were "working _on it_ now". :-)
